pkg/apis/greenhouse/v1alpha1/pluginconfig_types.go

```diff
@@ -26,6 +26,10 @@ type PluginConfigSpec struct {
 
 	// ClusterName is the name of the cluster the pluginConfig is deployed to. If not set, the pluginConfig is deployed to the greenhouse cluster.
 	ClusterName string `json:"clusterName,omitempty"`
+
+	// ReleaseNamespace is the namespace in the remote cluster to which the backend is deployed.
+	// Defaults to the organization namespace if not set.
```

Comment:
This accommodates the planned change to use a special name for the managed namespace

```suggestion
	// Defaults to the Greenhouse managed namespace if not set.
```